sdk/parents/azure-client-sdk-parent/pom.xml

```diff
@@ -985,19 +985,25 @@
                   <release>${java.vm.specification.version}</release>
                 </configuration>
               </execution>
-              <!-- then compile with Java 11 as the base version that supports module-info -->
+
+              <!-- then compile with Java 9 as the base version that supports module-info -->
               <!-- this is the Azure SDK baseline for jars that are released to Maven and should only be updated
               when the Azure SDK policy is changed -->
-              <!-- executing this after default-compile will generate module-info with major class version 55 (i.e Java 11) -->
+              <!-- executing this after default-compile will generate module-info with major class version 53 (i.e Java 9) -->
               <execution>
                 <id>base-modules-compile</id>
                 <goals>
                   <goal>compile</goal>
                 </goals>
                 <configuration>
-                  <release>11</release>
+                  <!-- Use Java 9 to compile the module-info as we should support the lowest version above 8 available though most commonly this will be Java 11 as it's LTS -->
+                  <release>9</release>
+                  <includes>
```

Comment:
Few things
- Should we then add job that runs on java 9 into test matrix ?
- Do we compile core MRJAR with java 9 or 11?

Reply:
Forgot to answer the other question, I don't think we need to add a run with Java 9